polar/src/polar.lalrpop

```diff
@@ -120,79 +130,42 @@ Object<T>: BTreeMap<Symbol, Term> = {
     }
 };
 
-Dictionary<T>: Term = <start:@L> <fields:Object<T>> => {
+// ****** Strange Values ******* //
```

Comment:
What makes these "strange"?

Reply:
I can update this to something more meaningful. I found them strange because they aren't "pure" values - they contain expressions, and they are different whether its a term or a pattern.